src/main/java/jalau/at18/battlecity/Main.java

```diff
@@ -0,0 +1,31 @@
+package jalau.at18.battlecity;
+
+public class Main {
+    public static void main(String[] arg) {
+        Board board = createBoard();
+        showMatrix(board);
+    }
+
+    public static void showMatrix(Board board) {
+        Brick brick = new Brick();
+        Steel steel = new Steel();
+        Bushes bushes = new Bushes();
+
+        Element[][] matrix1 = board.getMatrix();
+
+        for (int row = 0; row < matrix1.length; row++) {
+            for (int column = 0; column < matrix1.length; column++) {
+                System.out.print(matrix1[row][column].getImage() + " ");
+            }
+            System.out.println();
+        }
+
+    }
+
+    public static Board createBoard() {
+        Stage stage = new Stage("C:\\Users\\maria\\AT18\\src\\main\\java\\jalau\\at18\\battlecity\\stage1.csv");
```

Comment:
please use relative path

Reply:
I will fix that in another pull request because I reverted that change